src/core/utils/compatibility.js

```diff
@@ -16,6 +16,14 @@ import { findInstallURL } from 'core/installAddon';
 import log from 'core/logger';
 
 
+// HACK: This is the ID on dev/stage/prod for the Facebook Container
+// add-on, which has a special-cased download URL supplied.
+// This should really be a special field set in the database that the front-end
+// can display, but this patch was last-minute and needed a timely turnaround.
+// See: https://github.com/mozilla/addons-server/issues/7982
+export const FACEBOOK_CONTAINER_ADDON_ID = 619864;
```

Comment:
Could this be keyed by guid instead? Won't using the id mean it won't work on -dev or stage?

Reply:
Oh, snap, sure. I thought the ID was the same everywhere on IRC but I misunderstood. Easily fixed.